pkg/icingadb/types/notification_type.go

```diff
@@ -15,17 +15,12 @@ type NotificationType uint16
 func (nt *NotificationType) UnmarshalText(text []byte) error {
 	s := string(text)
 
-	i, err := strconv.ParseUint(s, 10, 64)
+	i, err := strconv.ParseUint(s, 10, 16)
 	if err != nil {
 		return types.CantParseUint64(err, s)
 	}
```

Comment:
> CantParse**Uint64**

That will create a misleading error message: https://github.com/Icinga/icinga-go-library/blob/bb5b3747011a3d20befc17dafa0757f2eb3eb1e6/types/utils.go#L32-L35

There are no corresponding functions for other sizes than 64 so far unfortunately. :(